docs/examples/clipping.py

```diff
@@ -0,0 +1,74 @@
+"""
+Clipping Detection
+===================
+
+Identifying clipping periods using the PVAnalytics clipping module.
+"""
+
+# %%
+# Identifying and removing clipping periods from AC power time series
+# data aids in generating more accurate degradation analysis results,
+# as using clipped data can lead to over-predicting degradation. In this
```

Comment:
Under-predicting, I would think.

Reply:
Changed to under-predicting in 
commit 4f379e6e77b965fd72abbe33c477ab7c41d7a153